### PR TITLE
Update wording from directory to folder in user interface

### PR DESCRIPTION
**Pull request by Rob-P-Smith**
Adjusted wording to satisfy issue #16321

## References

This addresses rewording requested in #16231

## Code changes

Adjusted the labeling on file browser display from 'directory' to 'folder' in /filbrowser/schema/browser.json

## User-facing changes

Changes seen here 
folder

## Backwards-incompatible changes

This should not cause any issues with backwards compatibility.

### CODE REVIEW

**Comment by Rob-P-Smith** [comment]
I must be missing something, but I can't see where I would add the label to this pull request to satisfy the Enforce PR label test.

**Comment by krassowski** [comment]
> I must be missing something, but I can't see where I would add the label to this pull request to satisfy the Enforce PR label test.

https://jupyterlab.readthedocs.io/en/stable/developer/contributing.html#tagging-issues-with-labels (maybe docs should also mention that it also applies to PRs? If you agree PR welcome!)

@meeseeksdev tag enhancement

**Comment by Rob-P-Smith** [comment]
Krassowski,
  Thanks for helping a new contributor figure it out; I must have skimmed right past that. Mea culpa for failing at RTFM.
  
  On to a more meaningful task now!

**Comment by krassowski** [comment]
I think we should also update documentation for consistency, in particular [docs/source/user/files.rst](https://github.com/jupyterlab/jupyterlab/blob/main/docs/source/user/files.rst#working-with-files) but possibly other pages in User Guide as well.

> I accidentally pushed without branching

No worries, we do squash merge anyways.

**Comment by Rob-P-Smith** [comment]
Fixed mangled commit from yesterday and adding please update snapshots for the bot.

**Comment by Rob-P-Smith** [comment]
please update snapshots

File: packages/apputils-extension/src/index.ts [hunk 1]
```diff
@@ -650,7 +650,7 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
     });
 
     commands.addCommand(CommandIDs.displayShortcuts, {
-      label: trans.__('Show Keyboard Shortcuts'),
+      label: trans.__('Show Keyboard Shortcuts...'),
```
Comment (krassowski) [inline packages/apputils-extension/src/index.ts:653]: This looks to have been carried over from another PR. Let's revert it and snapshots which changed as a consequence of this change.

```suggestion
      label: trans.__('Show Keyboard Shortcuts'),
```

File: galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json [hunk 2]
```diff
@@ -177,7 +177,7 @@
   },
   {
     "id": "apputils:display-shortcuts",
-    "label": "Show Keyboard Shortcuts",
+    "label": "Show Keyboard Shortcuts...",
```
Comment (krassowski) [inline galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json:180]: This looks to have been carried over from another PR.

```suggestion
    "label": "Show Keyboard Shortcuts",
```

File: galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json [hunk 3]
```diff
@@ -1586,7 +1586,7 @@
   },
   {
     "id": "notebook:close-and-shutdown",
-    "label": "Close and Shut Down Notebook",
+    "label": "Close and Shut Down Notebook...",
```
Comment (krassowski) [inline galata/test/documentation/commands.test.ts-snapshots/commandsList-documentation-linux.json:1589]:
```suggestion
    "label": "Close and Shut Down Notebook",
```

File: packages/notebook-extension/src/index.ts [hunk 4]
```diff
@@ -2417,7 +2417,7 @@ function addCommands(
     isEnabled
   });
   commands.addCommand(CommandIDs.closeAndShutdown, {
-    label: trans.__('Close and Shut Down Notebook'),
+    label: trans.__('Close and Shut Down Notebook...'),
```
Comment (krassowski) [inline packages/notebook-extension/src/index.ts:2420]:
```suggestion
    label: trans.__('Close and Shut Down Notebook'),
```

File: packages/notebook-extension/schema/export.json [hunk 5]
```diff
@@ -13,7 +13,7 @@
             "rank": 10,
             "submenu": {
               "id": "jp-mainmenu-file-notebookexport",
-              "label": "Save and Export Notebook As…"
+              "label": "Save and Export Notebook As"
```
Comment (krassowski) [inline packages/notebook-extension/schema/export.json:16]:
```suggestion
              "label": "Save and Export Notebook As…"
```

**Review by krassowski — COMMENTED** [review]
Let's revert unrelated changes

File: docker/start.sh
Comment (krassowski) [inline docker/start.sh:1]: Needs to be deleted.

Comment (krassowski) [reply docker/start.sh:1]: The file is not empty contrarily to what GitHub is showing. It needs reverting to the state from the `main` branch.

**Review by krassowski — APPROVED** [review]
Thank you @Rob-P-Smith

**Pull request merged**